sdk/go/pulumi/run.go

```diff
@@ -197,6 +199,8 @@ const (
 	EnvOrganization = "PULUMI_ORGANIZATION"
 	// EnvProject is the envvar used to read the current Pulumi project name.
 	EnvProject = "PULUMI_PROJECT"
+	// EnvPulumiProjectRoot is the envvar used to read the current Pulumi project root, location of Pulumi.yaml.
+	EnvPulumiProjectRoot = "PULUMI_PROJECT"
```

Comment:
PROJECT is already used in the line above
```suggestion
	EnvPulumiProjectRoot = "PULUMI_ROOT_DIRECTORY"
```
Or PROJECT_ROOT, but everywhere else in the system we call this "root directory" it will be very confusing if we start calling it just "project directory" in some places. This also applies to your changes for the other language runtimes.